DQMOffline/RecoB/interface/JetTagPlotter.h

```diff
@@ -45,17 +45,21 @@ class JetTagPlotter : public BaseBTagPlotter {
 
   // binning and bounds
   // 1) for 'efficiency' versus discriminator cut histos
-  int    discrBins  ;
-  double discrStart_ ;
-  double discrEnd_   ;
-  int	nBinEffPur_ ;
-  double startEffPur_ ; 
-  double endEffPur_ ; 
+  int discrBins;
+  double discrStart_;
+  double discrEnd_;
+  int nBinEffPur_;
+  double startEffPur_; 
+  double endEffPur_; 
 
   unsigned int mcPlots_;
   bool willFinalize_;
 
   bool doCTagPlots_;
+  
+  // Differential plots: efficiency vs. variable for cut on discrimator > cutValue_
+  bool doDifferentialPlots_;
+  double cutValue_;
 
   int *nJets;
```

Comment:
This is never `delete`d that I can see.
Please just use an `std::vector<int>` instead.